migrations/1723505340806-addNotificationTypesForQacc.ts

```diff
@@ -0,0 +1,236 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
```

Comment:
We don't need that much notification, as most of these events won't happen in QAcc.
The reference migration can be migrations/1660716115917-seedNotificationType.ts
From them, we may only need Made Donation, or something else that later we add.